mmv1/products/compute/terraform.yaml

```diff
@@ -831,16 +842,12 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           backend_service_name: "backend"
           igm_name: "igm-internal"
       - !ruby/object:Provider::Terraform::Examples
-        name: "private_service_connect_google_apis"
+        name: "global_forwarding_rule_private_services_connect"
```

Comment:
This name corresponds with the file name of the test config to use. The test config file will need to be renamed to correspond with this